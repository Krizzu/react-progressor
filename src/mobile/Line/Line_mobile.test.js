import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import LineProgressor from './Line';


it('Renders correctly', () => {
  const tree = renderer.create(
    <LineProgressor />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
