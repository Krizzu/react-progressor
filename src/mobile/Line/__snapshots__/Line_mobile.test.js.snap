// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "position": "relative",
      },
      Object {
        "height": undefined,
        "width": undefined,
      },
      Object {
        "height": 20,
        "width": 200,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
        },
        Object {
          "backgroundColor": "#eee",
        },
        Object {
          "height": 20,
          "width": 200,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "left": 0,
          "position": "absolute",
          "top": 0,
        },
        Object {
          "backgroundColor": "#8DE969",
        },
        Object {
          "height": 20,
          "width": 0,
        },
      ]
    }
  />
</View>
`;
